docs/guides/deploy.md

```diff
@@ -131,3 +131,64 @@ Then, decrypt the result:
 decrypted_result = client.decrypt(deserialized_result)
 assert decrypted_result == 49
 ```
+
+# Deployment of modules
+
+Deploying a [module](../compilation/modules.md#modules) follows the same logic presented above. Assuming a module compiled in the following way:
+
+<!--pytest-codeblocks:skip-->
+```python
+from concrete import fhe
+
+@fhe.module()
+class MyModule:
+    @fhe.function({"x": "encrypted"})
+    def inc(x):
+        return x + 1
+
+    @fhe.function({"x": "encrypted"})
+    def dec(x):
+        return x - 1
+
+inputset = list(range(20))
+configuration = fhe.Configuration(
+    parameter_selection_strategy="v0",
+    composable=True,
+)
+my_module = MyModule.compile(
+    {"inc": inputset, "dec": inputset},
+    configuration,
+)
+```
+
+The server can be extracted from the module and saved to a file:
+
+<!--pytest-codeblocks:skip-->
+```python
+my_module.server.save("server.zip")
+```
+
+The only noticeable difference with the workflow presented above is the `Client::encrypt`, `Client::decrypt` and `Server::run` methods must be given an extra `function_name` argument specifying the name of the targeted function.
```

Comment:
The only noticeable difference between the deployment of modules and the deployment of circuits is `Client::encrypt`. You must give an extra `function_name` argument that specifies the name of the targeted function to`Client::decrypt` and `Server::run` methods for the deployment of modules.

Reply:
I think the way I wrote the sentence misled you. All the three methods must be given an extra `function_name` argument. I tried an other formulation.